extension/source/QuillPage/Onboarding/ReviewSecretPhrasePanel.tsx

```diff
@@ -64,6 +64,18 @@ const ReviewSecretPhrasePanel: FunctionComponent<{
     setAllCorrect(reviewWordStates.every((item) => item === true));
   };
 
+  const navigate = useNavigate();
+
+  const setHDWalletPhrase = async () => {
+    // @ts-ignore
```

Comment:
I don't think there is a valid reason to ignore typescript here?

(There are lots of examples of this in this PR.)

Reply:
yeah, basically added `// @ts-ignore` wherever window object is used. This was done just to get the account generation working asap. The correct way to do would be to store the controller in state and pass it visa context/global state. Discussed this with @jacque006